test/tests/monica-apache-run/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+mysqlImage='mysql:5.7'
```

Comment:
The only feedback I have is a recent improvement to the tests (https://github.com/docker-library/official-images/pull/8040), but I can add it later. :smile_cat:

https://github.com/docker-library/official-images/blob/39708ad55fab9b12c0c21afbc0a57a78e2e02a24/test/tests/wordpress-apache-run/run.sh#L14-L17

Reply:
I can add it now if you prefer?

Reply:
Sure! :heart:

Reply:
... and done!